### DISCUSSION TITLE
SubTopics for different pages

### DISCUSSION THREAD

**Discussion by paule32** [doxygen/doxygen#11052] [General]
Hello,
how can I mix MarkDown Style, and display SubTopics in a extra Page at CHM Project ?
Currently, all SubTopics styled with ## appear on one Page

![grafik](https://github.com/user-attachments/assets/6e2fa1ff-f8c5-4b2b-a825-78f2d170fa53)

Here is a packed DoxyProject:
[packed.zip](https://github.com/user-attachments/files/16487864/packed.zip)

**Comment by albert-github** [comment]
Maybe use `\page`  (for "Aufzählungen", "Schleifen" etc.) and `\subpage` on the mainpage after the "Schlüsselwörter" or distribute the data over groups and use `DISTRIBUTE_GROUP_DOC=YES`?

**Reply by paule32** [reply]
ok, with \page I get it working.
But, when I use \defgroup, \ingroup, and @{ ... @} I stuck into a little Problem:
when I would documenting the grouping:

\defgroup page1 Pagename
@{ @}  /// End of Group:

The comment after @} will be evaluate, and rendered in the Doxygen Output.
Is it possible, to disable this, so comments (in the same line) after @} are out of scope of the documenting Texts ?
Because, it makes the maintain of the Documentation more user friendly, to have a "reminder" for/in which group scope the Text lay out.

![grafik](https://github.com/user-attachments/assets/3dd5623d-0a66-4a60-a7e5-a43860840bac)

**Comment by paule32** [comment]
I forget to give away a sample Project:
[packed.zip](https://github.com/user-attachments/files/16494481/packed.zip)

BTW: How can I add an Attachment, to a Reply, when I edit the posted comment - or have I do post a new Reply?

**Reply by albert-github** [reply]
My first impression was oops a bug, but this is not the case the `/// Gruppe: DO` is just normal input.
I see the following solutions, replace the line:
```
* @}   /// Gruppe: DO
```
by
```
* @}   */ /* Gruppe: DO */ /**
```
or
```
* @}   \noop Gruppe: DO
```


Small (unrelated) side note:
```
STRIP_FROM_PATH        = YES
STRIP_FROM_INC_PATH    = YES
```
these settings are not boolean settings but should contain the paths to be stripped.

> BTW: How can I add an Attachment, to a Reply, when I edit the posted comment - or have I do post a new Reply?

probably when in editing mode by dragging the file on edit space (I didn't test), but when there is some time between the message that should the attachment and the adding of the attachment it is better to make a new reply (otherwise it might go unnoticed as edits don't trigger email notifications).

**Reply by paule32** [reply]
thank you very much.
You made my day - it is working.
